src/main/java/org/aion/wallet/ui/components/SendController.java

```diff
@@ -138,6 +148,10 @@ protected void registerEventBusConsumer() {
         EventBusFactory.getBus(EventPublisher.ACCOUNT_CHANGE_EVENT_ID).register(this);
     }
 
+    private double convertBalanceToCcy(final AccountDTO account, final double exchangeRate) {
+        return Double.parseDouble(account.getBalance()) * exchangeRate;
```

Comment:
NPE?

Reply:
True that, but this is also the case wherever he have a `handleAccountChanged()` call so I modified the EventPublisher so it won't fire events with `null` accounts